_docs/tasks/security/mutual-tls.md

```diff
@@ -62,6 +65,13 @@ For example, after starting the [Bookinfo]({{home}}/docs/guides/bookinfo.html)
 sample application you can ssh into the Envoy container of `productpage` service,
 and send request to other services by curl.
 
+> By default istio proxy image does not have curl installed. To try this
```

Comment:
default istio proxy image -> default, the Istio proxy image

Reply:
Put curl inside backticks

Reply:
done